src/validation/rules/ValuesOfCorrectTypeRule.ts

```diff
@@ -216,6 +216,13 @@ function validateOneOfInputObject(
   if (isVariable) {
     const variableName = value.name.value;
     const definition = variableDefinitions[variableName];
+
+    // If the variable definition is missing, skip validation here.
+    // This should be caught by other validation rules.
+    if (!definition) {
+      return;
+    }
```

Comment:
Technically "Values of Correct Type" in the spec only applies to literals ("For each literal Input Value {value} in the document:"), what you're actually dealing with here is maybe "All Variable Usages Are Allowed"?

But yes, I think in general the spec probably needs all of its validation rules to say "if <X> doesn't exist, return" where that situation has already been covered by a different validation rule explicitly checking that X exists.